python/graphstorm/model/gnn_encoder_base.py

```diff
@@ -72,6 +74,80 @@ def layers(self):
         """
         return self._layers
 
+    def dist_inference(self, g, get_input_embeds, batch_size, fanout,
```

Comment:
Why we need to define this function?

Reply:
because the module for reconstructing node features belongs to the GNN encoder. The full-graph inferences with the normal GNN encoder and the GNN encoder with feature reconstruction are different.